### DISCUSSION TITLE
Volume control does not work on elementary OS 7.1 Horus

### DISCUSSION THREAD

**Discussion by beretar-tg** [elementary/settings#303] [Q&A]
The slider runs back and forth, but the volume does not change.
[diag.txt](https://github.com/elementary/switchboard/files/14613403/diag.txt)

**Comment by danirabbit** [comment]
Hey thanks for your feedback. Discussions are used to ask questions or to create open ended discussions. To report an issue, please file a new report here: https://github.com/elementary/switchboard-plug-sound/issues/new?assignees=&labels=&projects=&template=bug_report.yml
